app/templates/main/login.html

```diff
@@ -0,0 +1,52 @@
+{%- from 'govuk_frontend_jinja/components/button/macro.html' import govukButton -%}
+{%- from 'govuk_frontend_jinja/components/header/macro.html' import govukHeader -%}
+{%- from 'govuk_frontend_jinja/components/phase-banner/macro.html' import govukPhaseBanner -%}
+{%- from 'govuk_frontend_jinja/components/footer/macro.html' import govukFooter -%}
+
+{% block head %}
+  <meta name="description" content="{{ config['SERVICE_NAME'] }}">
+  <meta name="keywords" content="GOV.UK, govuk, gov, government, uk, frontend, ui, user interface, jinja, python, flask, port, template, templating, macro, component, design system, html, forms, wtf, wtforms, widget, widgets, demo, example">
+  <meta name="author" content="{{ config['DEPARTMENT_NAME'] }}">
+  <link rel="stylesheet" type="text/css" href="{{ url_for('static', filename='govuk-frontend-4.7.0.min.css') }}" />
+  {% assets "css" %}<link href="{{ ASSET_URL }}" rel="stylesheet">{% endassets %}
+{% endblock head %}
+
+{% block header %}
+  {{ govukHeader({
+    'homepageUrl': url_for('main.index'),
+    'serviceName': 'Submit monitoring and evaluation',
+    'serviceUrl': url_for('main.index'),
+  }) }}
+{% endblock header %}
+
+{% block content %}
+
+<main id="main-content" role="main">
+
+    <section class="govuk-main-wrapper">
+        <div class="govuk-width-container">
+        <h1 class="govuk-heading-xl">Sign in to upload your return data</h1>
+        <p class="govuk-body">To submit your monitoring and evaluation data return for Towns Fund you need to sign in using the same details you use to access Microsoft Sharepoint.</p>
```

Comment:
I know it's designs area, but SharePoint is not the right reference, should be microsoft account or email

Reply:
We can mention this to design :)